tests/helpers/has-global-yarn.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const which = require('which');
+
+function hasGlobalYarn() {
+  return which.sync('yarn', { nothrow: true });
+}
+
+module.exports = hasGlobalYarn;
```

Comment:
exporting a function here still means that we call `which` whenever we call the function. it would be best if we only did it once though.

I think something like 
```js
module.exports = which.sync('yarn', { nothrow: true });
```
should be sufficient here.